### ISSUE TITLE
Andrdoid only:  box-shadow: none     set on Button widget leads to crash

### ISSUE THREAD

**Issue by cjohn001** [NativeScript/NativeScript#10462]
### Issue Description

When setting the box shadow on a Button, the initial rendering of the Button results in a crash of my app. My CSS

.ns-root .icon-font-button {
	background-color: transparent;
	color: $info;
	border-width: 0;
	font-size: 24;
	box-shadow: none;
	border-color: transparent;
	android-elevation: 0;
	android-dynamic-elevation-offset: 0;
}

The stack

<img width="601" alt="Bildschirmfoto 2023-12-19 um 18 17 55" src="https://github.com/NativeScript/NativeScript/assets/43253483/aad5c312-8f52-40c0-8cab-9112866a6772">

### Environment

```yaml
OS: macOS 14.1.2
CPU: (10) arm64 Apple M1 Pro
Shell: /bin/zsh
node: 18.17.1
npm: 9.6.7
nativescript: 8.6.1

# android
java: 11.0.21
ndk: Not Found
apis: Not Found
build_tools: Not Found
system_images: Not Found

# ios
xcode: 15.1/15C65
cocoapods: 1.14.3
python: 3.11.6
python3: 3.11.6
ruby: 2.7.8
platforms: 
  - DriverKit 23.2
  - iOS 17.2
  - macOS 14.2
  - tvOS 17.2
  - watchOS 10.2
```

### Dependencies

```json
"dependencies": {
  "@angular/animations": "16.2.11",
  "@angular/common": "16.2.11",
  "@angular/compiler": "16.2.11",
  "@angular/core": "16.2.11",
  "@angular/forms": "16.2.11",
  "@angular/platform-browser": "16.2.11",
  "@angular/platform-browser-dynamic": "16.2.11",
  "@angular/router": "16.2.11",
  "@apollo/client": "3.8.8",
  "@mnd/external-web-view": "file:../app-plugins/dist/packages/external-web-view/mnd-external-web-view-1.0.1.tgz",
  "@nativescript/angular": "16.0.1",
  "@nativescript/core": "8.6.1",
  "@nativescript/iqkeyboardmanager": "2.1.1",
  "@nativescript/localize": "5.2.0",
  "@nativescript/mlkit-barcode-scanning": "file:../app-plugins-customized/mlkit/dist/packages/mlkit-barcode-scanning/nativescript-mlkit-barcode-scanning-2.0.0.tgz",
  "@nativescript/mlkit-core": "file:../app-plugins-customized/mlkit/dist/packages/mlkit-core/nativescript-mlkit-core-2.0.0.tgz",
  "@nativescript/secure-storage": "3.0.3",
  "@nativescript/theme": "3.0.2",
  "@nativescript/ui-charts": "0.2.4",
  "apollo-angular": "5.0.2",
  "apollo3-cache-persist": "0.14.1",
  "d3-ease": "3.0.1",
  "qs": "npm:querystring@0.2.1",
  "graphql": "16.8.1",
  "graphql-tag": "2.12.6",
  "intl": "1.2.5",
  "moment": "2.29.4",
  "nativescript-health-data": "file:../app-plugins-customized/nativescript-health-data/publish/package/nativescript-health-data-2.0.0.tgz",
  "nativescript-oauth2-ext": "file:../app-plugins-customized/nativescript-oauth2-ext/publish/package/nativescript-oauth2-ext-3.0.3.tgz",
  "nativescript-sqlite": "2.8.6",
  "nativescript-sqlite-commercial": "file:../app-plugins-customized/commercial-sqlite/nativescript-sqlite-commercial-1.8.0.tgz",
  "nativescript-sqlite-encrypted": "file:../app-plugins-customized/commercial-sqlite/nativescript-sqlite-encrypted-1.6.0.tgz",
  "nativescript-ui-calendar": "15.2.3",
  "nativescript-ui-gauge": "15.2.3",
  "rxjs": "7.8.1",
  "util": "0.12.5",
  "uuidjs": "5.0.1",
  "zone.js": "0.14.2"
},
"devDependencies": {
  "@angular-devkit/build-angular": "16.2.8",
  "@angular/compiler-cli": "16.2.11",
  "@graphql-codegen/cli": "5.0.0",
  "@graphql-codegen/fragment-matcher": "5.0.0",
  "@graphql-codegen/introspection": "4.0.0",
  "@graphql-codegen/typescript": "4.0.1",
  "@graphql-codegen/typescript-apollo-angular": "4.0.0",
  "@graphql-codegen/typescript-operations": "4.0.1",
  "@nativescript/android": "8.6.2",
  "@nativescript/ios": "8.6.3",
  "@nativescript/types": "8.6.1",
  "@nativescript/webpack": "5.0.18",
  "@ngtools/webpack": "17.0.7",
  "@types/d3-ease": "3.0.2",
  "@types/intl": "1.2.2",
  "@types/node": "20.10.5",
  "keycloak-request-token": "0.1.0",
  "rimraf": "5.0.5",
  "sass": "1.69.5",
  "ts-node": "10.9.2",
  "typescript": "4.9.5"
}
```

Note: I have just downgraded from Angular 17 where I was also seeing the bug. I downgraded as I expected this to be the problem. However, this is not the case. Seems to be a core issue which only appears on Android

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by cjohn001** [comment]
Update: The bug is fixed in 8.6.2-next-12-19-2023-7265848970

I will close the issue once I was able to verify against 8.6.2.

**Comment by cjohn001** [comment]
fixed with 8.6.2
